src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -3,612 +3,753 @@
+"""
+
 helps['sql db replica delete-link'] = """
-    type: command
-    short-summary: Permanently stop data replication between two database replicas.
-    """
-helps['sql db export'] = """
-    type: command
-    short-summary: Export a database to a bacpac.
-    examples:
-        - name: Get an SAS key for use in export operation.
-          text: |
-            az storage blob generate-sas --account-name myAccountName -c myContainer -n myBacpac.bacpac \\
-                --permissions w --expiry 2018-01-01T00:00:00Z
-        - name: Export bacpac using an SAS key.
-          text: |
-            az sql db export -s myserver -n mydatabase -g mygroup -p password -u login \\
-                --storage-key "?sr=b&sp=rw&se=2018-01-01T00%3A00%3A00Z&sig=mysignature&sv=2015-07-08" \\
-                --storage-key-type SharedAccessKey \\
-                --storage-uri https://myAccountName.blob.core.windows.net/myContainer/myBacpac.bacpac
-        - name: Export bacpac using a storage account key.
-          text: |
-            az sql db export -s myserver -n mydatabase -g mygroup -p password -u login \\
-                --storage-key MYKEY== --storage-key-type StorageAccessKey \\
-                --storage-uri https://myAccountName.blob.core.windows.net/myContainer/myBacpac.bacpac
-    """
-helps['sql db import'] = """
-    type: command
-    short-summary: Imports a bacpac into an existing database.
-    examples:
-        - name: Get an SAS key for use in import operation.
-          text: |
-            az storage blob generate-sas --account-name myAccountName -c myContainer -n myBacpac.bacpac \\
-                --permissions r --expiry 2018-01-01T00:00:00Z
-        - name: Import bacpac into an existing database using an SAS key.
-          text: |
-            az sql db import -s myserver -n mydatabase -g mygroup -p password -u login \\
-                --storage-key "?sr=b&sp=rw&se=2018-01-01T00%3A00%3A00Z&sig=mysignature&sv=2015-07-08" \\
-                --storage-key-type SharedAccessKey \\
-                --storage-uri https://myAccountName.blob.core.windows.net/myContainer/myBacpac.bacpac
-        - name: Import bacpac into an existing database using a storage account key.
-          text: |
-            az sql db import -s myserver -n mydatabase -g mygroup -p password -u login --storage-key MYKEY== \\
-                --storage-key-type StorageAccessKey \\
-                --storage-uri https://myAccountName.blob.core.windows.net/myContainer/myBacpac.bacpac
-    """
+type: command
+short-summary: Permanently stop data replication between two database replicas.
+"""
+
+helps['sql db replica list-links'] = """
+type: command
+short-summary: List the replicas of a database and their replication status.
+examples:
+  - name: List the replicas of a database and their replication status. (autogenerated)
+    text: az sql db replica list-links --name MyAzureSQLDatabase --resource-group MyResourceGroup --server myserver
```

Comment:
Instead of MyAzureSQLDatabase, please generate something shorter like mydb. The user is already in the context of Azure SQL DB due to the name of the command, we don't need to beat them over the head with it.